website/docs/r/automation_schedule.html.markdown

```diff
@@ -46,17 +47,19 @@ The following arguments are supported:
 
 * `resource_group_name` - (Required) The name of the resource group in which the Schedule is created. Changing this forces a new resource to be created.
 
-* `account_name` - (Required) The name of the automation account in which the Schedule is created. Changing this forces a new resource to be created.
```

Comment:
Since we do not have versioned documentation at the moment, not sure if we should leave the old argument documented with **deprecated** and noting the new name to reduce any confusion